src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -834,6 +834,11 @@ public function leaveHash(Cursor $cursor, $type, $class, $hasChild, $cut)
         parent::leaveHash($cursor, $type, $class, $hasChild, 0);
     }
 
+    public function dumpImage(Cursor $cursor, $str, $bin, $cut)
+    {
+        $this->line .= sprintf('<img src=data:image/png;base64,%s />', base64_encode($str));
```

Comment:
Should we add some max-width + max-height constraints to protect against huge images? After all, we already "trim" long strings (we don't trim, we just hide the rest of the long string) so it may make sense to do the same here. Thanks.